gateway/handler_error.go

```diff
@@ -23,6 +23,8 @@ const (
 	defaultContentType    = headers.ApplicationJSON
 )
 
+var customizableErr = errors.New("customizable error")
```

Comment:
I think this error should have a better name (and error text) as it will be hard to remember in 6 months what the purpose of this error was. I could even think about it to add an explaining comment to it.

something like `errCustomBodyResponse` could work.